src/murmur/main.cpp

```diff
@@ -16,6 +16,9 @@
 #include "ServerDB.h"
 #include "Version.h"
 
+#include <boost/logic/tribool.hpp>
+#include <boost/optional/optional_io.hpp>
+#include <boost/tuple/tuple_io.hpp>
```

Comment:
If possible, I would like to stick to std-types. If that would make things a lot more complicated, stick to using Boost though.

Reply:
I tried using std types and experienced some issues since they are not always available depending on which target system is built. Yeah boost is quite expensive to compile, I agree - not using them would be better. I'll try to see if I can remove them after things work

Reply:
std-types should always present. You only have to check to stick to std types included in up to cpp14 as that's the current standard we are using.

Reply:
Sadly, std::optional is not avalable in cpp14 afaik. I'll try to find another solution.

Reply:
Yes, but an empty string will do equally fine for indicating that the INI file has not been specified :)